mcs/class/System.ComponentModel.Composition.4.5/Makefile

```diff
@@ -14,7 +14,7 @@ KEYFILE = ../ecma.pub
 LIB_MCS_FLAGS = -d:CLR40 -d:USE_ECMA_KEY,FEATURE_REFLECTIONCONTEXT,FEATURE_REFLECTIONFILEIO,FEATURE_SERIALIZATION,FEATURE_SLIMLOCK -nowarn:219,414 -nowarn:436
 
 XTEST_LIB_REFS = System System.Core System.Xml Facades/System.Threading.Tasks
-XTEST_LIB_FLAGS := /keyfile:../mono.snk
+XTEST_LIB_FLAGS := /keyfile:../mono.snk -publicsign
```

Comment:
😭 this is a regression in Roslyn 2.8. Full signing worked fine with 2.7.0.62620 (e873e693)

Reply:
Tracked it down: https://github.com/dotnet/roslyn/issues/26678